files/build_templates/sonic_debian_extension.j2

```diff
@@ -724,6 +724,14 @@ sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install {{ name }}=={{
 sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install --from-tarball {{ path }} {{ get_install_options(set_owner, enabled) }}
 {% endfor -%}
 
+{% if install_third_party_debs -%}
+sudo LANG=C chroot $FILESYSTEM_ROOT bash -c '
+TEMP_DIR="$(mktemp -d);
+wget {{ install_third_party_debs.strip().replace(",", " ") }} --directory-prefix="$TEMP_DIR" || exit 1;
+dpkg --install --recursive "$TEMP_DIR" || exit 1
```

Comment:
@stepanblyschak do we also need `|| sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install -f` ?

Reply:
This hides errors. I changed to "apt-get install -y <dir>/*"